### DISCUSSION TITLE
MRU program list in OpenShell is blank

### DISCUSSION THREAD

**Discussion by tsherrington** [Open-Shell/Open-Shell-Menu#1592] [General]
I've seen this on a few computers now.  When I click the start button, the left hand side, which normally shows the MRU program list, is completely blank.  Is there a way to fix this or reset it so it starts filling in again?

Thanks,

Thane

**Comment by ge0rdi** [comment]
Make sure you have `Recent programs` selected in `Open-Shell Settings` -> `Main menu`  -> `Show recent or frequent programs`.
